modules/chapter2/pages/section4.adoc

```diff
@@ -2,6 +2,7 @@
 :navtitle: Lab
 
 In this exercise, you will practice the use of data science projects, permissions, workbenches, environment variables, and data connections.
+//Need more information on the objectives and goals of the lab.
```

Comment:
hmm - maybe the objective is to learn these basic concepts of projects, permissions, workbenches, env vars and connections? please suggest alternative objectives